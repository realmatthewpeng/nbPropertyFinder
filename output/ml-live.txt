Total number of lines found: 21

tollrides = trips[trips['tolls_amount'] > 0]
tollrides[tollrides['pickup_datetime'] == '2012-09-05 15:45:00']
trips.describe()
tripsqc.describe()
trips = trips_in.copy(deep=True)
del trips['tolls_amount']
del trips['pickup_datetime']
del trips['trip_distance']
del trips['total_amount']
df_train = shuffled.iloc[:trainsize, :]
df_valid = shuffled.iloc[trainsize:(trainsize+validsize), :]
df_train.describe()
df_valid.describe()
rate = df_train['fare_amount'].mean() / estimate_distance(df_train).mean()
return np.sqrt(np.mean((actual-predicted)**2))
FEATURES = CSV_COLUMNS[1:len(CSV_COLUMNS) - 1]
LABEL = CSV_COLUMNS[0]
os.environ['BUCKET'] = BUCKET
os.environ['REGION'] = REGION
os.environ['TFVERSION'] = '1.8'
os.environ['PROJECT'] = PROJECT

Incorrect Lines:

return np.sqrt(np.mean((actual-predicted)**2))
FEATURES = CSV_COLUMNS[1:len(CSV_COLUMNS) - 1]
LABEL = CSV_COLUMNS[0]
os.environ['BUCKET'] = BUCKET
os.environ['REGION'] = REGION
os.environ['TFVERSION'] = '1.8'
os.environ['PROJECT'] = PROJECT

Precision = 0.6666666666666666
Recall = 0.5384615384615384

trips[:10]
trips['pickup_longitude'] > -78, \
trips['pickup_longitude'] < -70, \
trips['dropoff_longitude'] > -78, \
trips['dropoff_longitude'] < -70, \
trips['pickup_latitude'] > 37, \
trips['pickup_latitude'] < 45, \
trips['dropoff_latitude'] > 37, \
trips['dropoff_latitude'] < 45, \
trips['passenger_count'] > 0,
tripsqc = preprocess(trips)
cols = outdf.columns.tolist()
