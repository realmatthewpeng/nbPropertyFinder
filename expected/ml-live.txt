trips[:10]
tollrides = trips[trips['tolls_amount'] > 0]
tollrides[tollrides['pickup_datetime'] == '2012-09-05 15:45:00']
trips.describe()
trips = trips_in.copy(deep=True)
del trips['tolls_amount']
del trips['total_amount']
del trips['trip_distance']
del trips['pickup_datetime']
trips['pickup_longitude'] > -78, \
trips['pickup_longitude'] < -70, \
trips['dropoff_longitude'] > -78, \
trips['dropoff_longitude'] < -70, \
trips['pickup_latitude'] > 37, \
trips['pickup_latitude'] < 45, \
trips['dropoff_latitude'] > 37, \
trips['dropoff_latitude'] < 45, \
trips['passenger_count'] > 0,
tripsqc = preprocess(trips)
tripsqc.describe()
df_train.describe()
df_valid.describe()
cols = outdf.columns.tolist()